js/docs/src/content/docs/guides/streamplace-dev-setup.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Getting Started with Streamplace Development
+description: Learn how to set up your development environment for Streamplace.
+---
+
+So, you've decided to contribute to Streamplace development. Here's how you can
+get started:
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/)
+- [Yarn](https://yarnpkg.com/)
+  - A way to install it is with `pnpm/npm install -g yarn`. No corepack in this
```

Comment:
Huh, I totally thought we did. Maybe I just had corepack on one of the dev computers and it installed it anyway? PRs welcome!